### PR TITLE
Remove "-d" short flag from "delivery-type" and "days-ago" options

### PR DESCRIPTION
**Pull request by eliottBo**
## Description
Fixing issue #3234
Remove "-d" short flag from "delivery-type" and "days-ago" option to avoid conflict with "-d" from dry-run option.

### Changed

- Remove "-d" short flag from "delivery-type" option in ```/Users/eliottbosshard/cg/cg/cli/deliver/base.py``` and "days-ago" option in ```/Users/eliottbosshard/cg/cg/cli/upload/mutacc.py```

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [X] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by eliottBo** [comment]
The following documentation will need to be changed as well in Atlas:
- [ ]  https://atlas.scilifelab.se/production/data_analysis/workflows/microsalt/
- [ ]  /Users/eliottbosshard/atlas/docs/production/data_analysis/supporting_information/supportsystem_usage.md
- [ ]  /Users/eliottbosshard/atlas/docs/production/data_analysis/supporting_procedures/delivery.md
- [ ]  /Users/eliottbosshard/atlas/docs/production/data_analysis/workflows/balsamic.md
- [ ]  /Users/eliottbosshard/atlas/docs/production/data_analysis/workflows/microsalt.md
- [ ]  /Users/eliottbosshard/atlas/docs/production/data_analysis/workflows/raw_data_delivery.md
- [ ]  /Users/eliottbosshard/atlas/docs/production/data_analysis/workflows/sars-cov-2.md

**Review by ivadym — APPROVED** [review]
Nice 👍 

Great work identifying the affected atlas documents 💯

**Review by henrikstranneheim — APPROVED** [review]
👍

**Comment by diitaz93** [comment]
I spoke with @Karl-Svard and production would prefer removing the `-d` of dry-run instead, as they use the `-d` of delivery on a daily basis almost

**Comment by henrikstranneheim** [comment]
Ok, sounds good! Usability is key here I think.

**Comment by eliottBo** [comment]
Removing the -d flag would also mean to change the tests in place.
Isn't @ivadym's suggestion to change the delivery-type flag to "-dt" and the days-ago flag to "-da" the easiest way? In this way they are still short and easy to use in production and all we have is to change the documentation.
(Just a suggestion)

**Comment by ivadym** [comment]
> Removing the -d flag would also mean to change the tests in place. Isn't @ivadym's suggestion to change the delivery-type flag to "-dt" and the days-ago flag to "-da" the easiest way? In this way they are still short and easy to use in production and all we have is to change the documentation. (Just a suggestion)

Since we'll be deprecating the data delivery flag, I would say we remove the `-d` from the dry run and avoid messing with prod bioinfo. I also think it makes sense to be more explicit when calling the `--dry-run`, and on top of that the short version is usually `-n`, not `-d`.

File: cg/models/cg_config.py [hunk 1]
```diff
@@ -301,27 +301,6 @@ class DataFlowConfig(BaseModel):
...
-    flow_cell_runs_dir: str
-    demultiplexed_runs_dir: str
-
-
-class RunInstruments(BaseModel):
-    pacbio: PacbioConfig
-    nanopore: OxfordNanoporeConfig
-    illumina: IlluminaConfig
-
-
```
Comment (ChrOertlin) [inline cg/models/cg_config.py:324]: I think you need to check your branch. This undoes some changes introduced with other prs

Comment (eliottBo) [reply cg/models/cg_config.py:324]: Yes I had a problem with git, I created another branch (Remove dryrun flag #3244) to replace this one.

Comment (ivadym) [reply cg/models/cg_config.py:324]: Feel free to close this one to avoid confusion :)

**Comment by diitaz93** [comment]
Closing, replaced by #3244

**Pull request closed without merging**